packages/xod-client/src/project/components/layers/GhostsLayer.jsx

```diff
@@ -74,7 +74,7 @@ GhostLayer.displayName = 'GhostLayer';
 
 GhostLayer.propTypes = {
   mousePosition: PropTypes.objectOf(PropTypes.number),
-  mode: PropTypes.object, // eslint-disable-line
+  mode: PropTypes.string, // eslint-disable-line
```

Comment:
eslint complains here because it can't fugure out that prop is used.
I added exact rule to avoid confusion.